.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+@apsummers @thundercat1 @alfajor
```

Comment:
This is to enable default reviewers so I don't keep forgetting to add people.

Reply:
Should I change you to scb-mchaines or do you want thundercat1?

Reply:
Nah you're good, I just forgot to switch accounts after doing those code assessments